### PR TITLE
Set the cloudsource to new version after the upgrade

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

**Comment by jsuchome** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;BfMFXB3r&#x2F;64-5-s47p5-s48p19-add-ci-jobs-for-new-upgrade-process-with-and-without-ha"> (5) S47P5 (S48P19) Add CI jobs for new upgrade process with and without HA</a>

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3889,7 +3889,7 @@ function oncontroller_testsetup()
     wait_for 200 5 "openstack image show $imageid | grep status.*active" \
         "$image_name image to reach active state"
 
-    if [[ $want_ldap ]] ; then
+    if [[ $want_ldap = 1]] ; then
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Note: This is a workaround until https://github.com/SUSE-Cloud/automation/pull/1247 is tested and merged.

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: But it still should be there, I think.

Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: I think we do `[[ $want_something ]]` in other places as well. I agree with you though that this way of writing the tests is error prone and should be avoided. Your expression is a lot more clear.

**Pull request merged**